pandas/_libs/groupby.pyx

```diff
@@ -382,5 +383,55 @@ def group_any_all(uint8_t[:] out,
                 out[lab] = flag_val
 
 
+@cython.wraparound(False)
+@cython.boundscheck(False)
+def _group_add(floating[:, :] out,
```

Comment:
Unrelated to this PR but I see there are some functions at the end of the template file which have been converted to fused types but still remain in the template file. Could make sense to move them here as well (separate PR)